doc/compiling/linux.md

```diff
@@ -33,7 +33,7 @@ For openSUSE users:
 
 For Arch users:
 
-    sudo pacman -S --needed base-devel libcurl-gnutls cmake libpng sqlite libogg libvorbis openal freetype2 jsoncpp gmp luajit leveldb ncurses zstd gettext sdl2
+    sudo pacman -S --needed base-devel libcurl-gnutls cmake libpng libjpeg sqlite libogg libvorbis openal freetype2 jsoncpp gmp luajit leveldb ncurses zstd gettext sdl2
```

Comment:
should be "libjpeg-turbo"
according to https://archlinux.org/todo/stop-relying-on-virtual-libjpeg-lib32-libjpeg-package/